src/sparseml/integration_helper_functions.py

```diff
@@ -32,10 +32,14 @@ class Integrations(Enum):
     """
 
     image_classification = "image-classification"
+    transformers = "transformers"
+    transformers_generative = "transformers-generative"
```

Comment:
for top level integration - can we keep it as `transformers` and imply generative based on task name? this seems a bit cumbersome and hidden from UX perspective

Reply:
Let's discuss it today. 

I agree that it would be more appropriate to "fold" the `transformers_generative` under `transformers` in this hierarchy of argument. The contrary,however, still feels like a good implementation for one important reason: in our context, `transformers_generative` are quite distinct from the rest of `transformers`. They have their own optimizations, potentially recipe loading logic (need to look into it), different set of files. There is also high probability that the more we focus on LLMs, and the more popularity they gain, over time they will more-and-more become a separate concept from the "normal" transformers.

This is why I feel that the separate `IntegrationHelperFunctions` for `generative_transformers` makes sense, and from that follows the separate integration name.